cmd/libguestfs/entrypoint.sh

```diff
@@ -1,10 +1,11 @@
-#!/bin/bash -xe
+#!/bin/bash
 
-LIBGUESTFS_PATH=${LIBGUESTFS_PATH:=/tmp/guestfs}
-LIBGUEST_APPLIANCE=/usr/local/lib/guestfs/downloaded
-mkdir -p ${LIBGUESTFS_PATH}
-tar -Jxf ${LIBGUEST_APPLIANCE} -C ${LIBGUESTFS_PATH} --strip-components=1
```

Comment:
Yes, I need to try and check if this isn't  required by libguestfs-tools. But a good hint, thanks!

Reply:
Never mind, libguestfs drags in tar anyway :(

It's still good to drop the reference from `hack/rpm-deps.sh`, as that might change in the future, but maybe point out in the commit message that the package will still be included in the image.

And you can leave out the last commit, since there are no changes to be applied outside of the usual churn.